chart/flux/templates/helm-tls.yaml

```diff
@@ -1,5 +1,6 @@
 {{- if .Values.helmOperator.tls.enable -}}
 {{- if .Values.helmOperator.tls.verify -}}
+{{- if .Values.helmOperator.tls.caContent }}
```

Comment:
Can you please change this to a single if, with `if and `

Reply:
Hmm... 
I've just tested 3 and 4 - both seem to work ok. 

```
helm template --set helmOperator.tls.enable=true --set helmOperator.tls.verify=true --set helmOperator.tls.caContent="test" ./ -x /home/u2/Documents/repos/opensource/flux/chart/flux/templates/helm-tls.yaml
```

Reply:
3 looks ok to me, if that works please make the change

Reply:
Not sure why I thought 3 wouldn't work. I'll make the PR with number 3 as it seems to be working ok as outlined in command above.
@stefanprodan